src/main/java/zoe/Loader.java

```diff
@@ -0,0 +1,52 @@
+package zoe;
+
+import java.io.*;
+import java.util.ArrayList;
+
+public class Loader {
+    protected String path;
+    protected String fileName;
+
+    public Loader(String path, String fileName) {
+        this.path = path;
+        this.fileName = fileName;
+    }
+
+    /**
+     * Loads data from a saved file when zoe is opened again
+     * If no file exists, creates an empty Array list for zoe.
+     */
+    public ArrayList<Task> loadTasks(){
+        File file = new File(path, fileName);
+        try {
+            File directory = file.getParentFile();
+            //check for existence of directory
+            if (directory.exists() && file.exists()) {
+                BufferedReader reader = new BufferedReader(new FileReader(file));
+                ArrayList<Task> loadedTasks = new ArrayList<Task>();
+                String line;
```

Comment:
The guideline would suggest not creating the object without initialization, but I'm not sure how to write better here tbh.